app/views/users/show.html.slim

```diff
@@ -0,0 +1,17 @@
+p#notice
+  = notice
+p
+  strong
+    | Name:
+  = @user.user_profile.full_name
```

Comment:
Please have rather @user.full_name or in controller create @user_profile variable and then access data from it